uwblueprint/sistering
# Add Admin Posting Details Page

## Ticket link

Closes #155 



## Implementation description
* Refactored VolunteerPostingDetails page
* Created routing for AdminPostingDetails page
* Created query to fetch Posting data and the associated Admin Posting Details Page UI



## Steps to test
1. `docker-compose up --build`
2. Navigate to http://localhost:3000/admin/posting/1 and verify that the UI matches that specified in #155 



## What should reviewers focus on?
* Verify that the query works and the UI matches that specified in #155


## Checklist
